packages/default/scss/timeline/_layout.scss

```diff
@@ -354,7 +354,7 @@
                 z-index: 2;
 
                 .k-timeline-scrollable-wrap {
-                    transition: transform 1s ease-in-out;
+                    @include kendo-motion(short-2, ease-in-out, transform);
```

Comment:
I think the duration here should be _extra-long-4,_ not _short-2_ - please check